pkg/operations/kubernetesupgrade/upgrader.go

```diff
@@ -498,7 +501,7 @@ func (ku *Upgrader) upgradeAgentScaleSets(ctx context.Context) error {
 			ku.logger.Infof("Successfully set capacity for VMSS %s", vmssToUpgrade.Name)
 
 			// Before we can delete the node we should safely and responsibly drain it
-			client, err := ku.getKubernetesClient(cordonDrainTimeout)
```

Comment:
this used to be a const, we should be able to remove that const now that it's unused (or just turn it into the default)

Reply:
Yeah, I've removed the const.